Updates to extracting positions from sdf files

arunppsg opened the pull request:
This PR updates extracting molecular positions from qm9 dataset. Earlier, the `sdf` file was read as a text file and positions were extracted using patterns in the text file. Now, we use rdkit's utils directly to extract atom position in molecules from sdf files.

rusty1s reviewed (approved):
Cool, does that result in the same atom positions?

arunppsg commented:
Yes, here is a minimal example using `temp.sdf` with the following contents:
```
atom1


  5  4  0     0  0  0  0  0  0999 V2000
   -0.0127    1.0858    0.0080 C   0  0  0  0  0  0  0  0  0  0  0  0
    0.0022   -0.0060    0.0020 H   0  0  0  0  0  0  0  0  0  0  0  0
    1.0117    1.4638    0.0003 H   0  0  0  0  0  0  0  0  0  0  0  0
   -0.5408    1.4475   -0.8766 H   0  0  0  0  0  0  0  0  0  0  0  0
   -0.5238    1.4379    0.9064 H   0  0  0  0  0  0  0  0  0  0  0  0
  1  2  1  0  0  0  0
  1  3  1  0  0  0  0
  1  4  1  0  0  0  0
  1  5  1  0  0  0  0
M  END
$$$$
```

Extracting positions:
```
from rdkit import Chem
suppl = Chem.SDMolSupplier('temp.sdf', removeHs = False, sanitize=False)

for i, mol in enumerate(suppl):
    N = mol.GetNumAtoms()
    pos = suppl.GetItemText(i).split('\n')[4:4 + N]
    pos_via_text = [[float(x) for x in line.split()[:3]] for line in pos]

    conf = mol.GetConformer()
    pos_via_rdkit = conf.GetPositions()
```
Verifying
```
>>> print (pos_via_text)
[[-0.0127, 1.0858, 0.008],
 [0.0022, -0.006, 0.002],
 [1.0117, 1.4638, 0.0003],
 [-0.5408, 1.4475, -0.8766],
 [-0.5238, 1.4379, 0.9064]]

>>> print (pos_via_rdkit)
array([[-1.2700e-02,  1.0858e+00,  8.0000e-03],
       [ 2.2000e-03, -6.0000e-03,  2.0000e-03],
       [ 1.0117e+00,  1.4638e+00,  3.0000e-04],
       [-5.4080e-01,  1.4475e+00, -8.7660e-01],
       [-5.2380e-01,  1.4379e+00,  9.0640e-01]])
```

rusty1s commented:
Thanks for confirming:)
